edubots_surveys.Rnw

```diff
@@ -196,6 +196,8 @@
 \begin{abstractbox}
 
 \begin{abstract} % abstract
+Introducing new technologies such as messaging platforms, and the chatbots attached to them, in higher education, is rapidly growing. This introduction entails a careful consideration of the potential opportunities and/or challenges of adopting these tools. Hence, a thorough examination of the teachers’ experiences in this discipline can shed the light on the effective ways of enhancing students’ learning and boosting their progress. In this contribution, we have surveyed the opinions of tertiary education teachers based mainly in Spain and Spanish-speaking countries. The focus of these surveys is to collect teachers’ feedback about their opinions of the introduction of the messaging platforms and chatbots in their classes, understand their needs and to gather information about the various educational use cases where these tools are valuable. In addition, an analysis of how and when teachers’ attitudes towards the use of these tools can vary across gender, experience, and their discipline of specialisation is presented. The key findings of this study highlight the factors that can contribute to the advancement of the adoption of messaging platforms and chatbots in higher education institutions to achieve the desired learning outcomes.
```

Comment:
I am sorry, @NoorhanAbbas , but this really makes it difficult for me to compare and see the actual changes. If you simply substitute the old text with the new one, it will highlight whatever words and sentences are changed, and I'll be able to just approve it or make specific suggestions. This way, I need to do that by hand, creating two different files and comparing them using any comparation tool. Since I can edit your PR I'll do it right here, please check it out and try and do the rest of the PRs in the same way.